tests/handlers/aws/test_logstash_output.py

```diff
@@ -128,3 +137,32 @@ def test_sent_messages(self) -> None:
 
         msgs = self.logstash.get_messages()
         assert len(msgs) == 2
+
+    def test_failure_sending_messages(self) -> None:
+        os.environ["S3_CONFIG_FILE"] = _prepare_config_file(
+            self,
+            "config.yaml",
+            dict(CloudwatchLogStreamARN=self.cloudwatch_group_arn, LogstashURL="http://fake.url"),
+            "folder/config2.yaml",
+        )
+
+        event_cloudwatch_logs, event_ids_cloudwatch_logs = _logs_retrieve_event_from_cloudwatch_logs(
+            self.logs_client, group_name=self.group_name, stream_name=self.stream_name
+        )
+
+        ctx = ContextMock(TIMEOUT_15m)
+        handler(event_cloudwatch_logs, ctx)  # type: ignore
+        # test new input => output to stdout
+
+        msgs = self.logstash.get_messages()
+        assert len(msgs) == 0
+
+        sqs = self.aws_session.resource("sqs", endpoint_url=self.localstack.get_url())
+        queue = sqs.Queue(os.environ["SQS_REPLAY_URL"])
+        count = 0
+        for message in queue.receive_messages():
```

Comment:
Based on [boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.Client.receive_message), we should set "MaxNumberOfMessages" to a number, as the default is 1. Alternatively we could change that `for` with a `while`. Otherwise we would get only one message returned by the API.